File: Contributors/MartinRosenberg/twoSum/twoSum.js
#!/usr/bin/env node

const twoSum = (nums, target) => {
  const { length } = nums
  for (let i = 0; i < length; i++) {
    for (let j = i + 1; j < length; j++) {
      if (nums[i] + nums[j] === target) {
        return [i, j]
      }
    }
  }
}

File: Coding-Challenges/twoSum/solutions/two_sum.py
#!/usr/bin/env python3

def two_sum(nums, target):
    """
    :type nums: List[int]
    :type target: int
    :rtype: List[int]
    """
    for i in range(len(nums)):
        mid = target - nums[i]
        for j in range(i + 1, len(nums)):
            if nums[j] == mid:
                return [i, j]


print(two_sum([2, 7, 11, 15], 9))  # [0, 1]
